src/main.rs

```diff
@@ -0,0 +1,31 @@
+use std::fs::read_to_string;
+
+use serde::Deserialize;
+
+mod client;
+mod models;
+mod repository;
+
+#[derive(Default, Deserialize)]
+struct Configuration {
+    environment: String,
+    version: String,
+    api_key: String,
+}
+
+fn read_config(config_path: &str) -> std::io::Result<Configuration> {
+    let content = read_to_string(config_path)?;
+    let configuration = toml::from_str(&content)?;
+    Ok(configuration)
+}
```

Comment:
I'm thinking about taking the `Configuration` struct and the function `read_config` to a separate file. Thoughts?